iib/workers/tasks/build_merge_index_image.py

```diff
@@ -196,6 +197,7 @@ def _add_bundles_missing_in_source(
 @instrument_tracing(
     span_name="workers.tasks.build.handle_merge_request", attributes=get_binary_versions()
 )
+@Opm.set_opm
```

Comment:
I am not convinced by the idea of having a decorator inside the class. May I ask what's stopping us from having a single @classmethod inside the class like `set_opm_version` which will set the class variable if `None` otherwise return the class variable as is?